### DISCUSSION TITLE
Is there a keymapping can select mutiple file?

### DISCUSSION THREAD

**Discussion by WhateverMO** [nvim-tree/nvim-tree.lua#1891] [Q&A] [answered]
Is there a keymapping can select mutiple file?
i want to select many files and then cut them to another dir.

**Comment by alex-courtis** [comment, marked as answer]
Bookmarks / bulk should suit your needs.

Use `m` to mark files then `bmv` to move them to your desired location.

**Reply by WhateverMO** [reply]
Thanks a lot, that's exactly what i want!
